packages/@aws-cdk/aws-kinesisfirehose/lib/delivery-stream.ts

```diff
@@ -183,6 +204,22 @@ export interface DeliveryStreamProps {
    * @default - a role will be created with default permissions.
    */
   readonly role?: iam.IRole;
+
+  /**
+   * Indicates the type of customer master key (CMK) to use for server-side encryption, if any.
+   *
+   * If `encryptionKey` is provided, this will be implicitly set to `CUSTOMER_MANAGED`.
+   *
+   * @default StreamEncryption.UNENCRYPTED.
```

Comment:
Is this strictly correct? If `encryptionKey` is not undefined, this will be `CUSTOMER_MANAGED`, right?

Also, we don't end these with a full stop:

```suggestion
   * @default StreamEncryption.UNENCRYPTED
```